README.md

```diff
@@ -1,42 +1,214 @@
-# Serverless Framework: Deploy on Scaleway Functions (plugin)
+# Serverless Framework: Deploy on Scaleway Functions
 
-Developed and maintained by `Scaleway's Serverless Team`.
+This plugin allows [Serverless Framework](https://serverless.com) users to deploy their functions and containers to Scaleway Functions with a simple `serverless deploy` 
```

Comment:
What is Scaleway Functions ? You should probably add some description of this (these) products

Reply:
`Scaleway Functions` is a Serverless (Function As A Service) Platform, provided by [Scaleway](https://scaleway.com) to allow end-users to develop and deploy serverless workloads (we manage your Ops, scalability, availability, while you pay only for your usage, when your code gets executed) on Scaleway's fully managed infrastructure.